crates/core/src/test_utils/deploy_helpers.rs

```diff
@@ -536,6 +536,35 @@ impl DeployedKakarot {
     }
 }
 
+/// Returns a `StarknetConfig` instance customized for Kakarot.
+pub fn get_kakarot_starknet_config() -> StarknetConfig {
```

Comment:
small Rust idiomatic moment (purely aesthetic) -> Rust getters method usually don't incorporate the word "get", for some reason

should we follow that rusty folklore?

Reply:
https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter